app/components/CartComponent.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { PropTypes, Component } from 'react'
+import { Table, Button } from 'react-bootstrap'
+
+export default class CartComponent extends Component {
+  constructor(props) {
+    console.log(props)
```

Comment:
Looks like you left this in by accident! Also, if you use the react dev tools you can check the props of your components in your browser without having to console.log. = ).